services/order.py

```diff
@@ -0,0 +1,39 @@
+from django.db import transaction
+from db.models import Order, Ticket, User, MovieSession
+from django.db.models import QuerySet
+from django.utils.dateparse import parse_datetime
+
+
+def create_order(tickets: list, username: str, date: str = None) -> None:
+    user = User.objects.get(username=username)
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user)
+
+        for ticket_data in tickets:
+            movie_session_id = ticket_data["movie_session"]
+            row = ticket_data["row"]
+            seat = ticket_data["seat"]
+
+            movie_session = MovieSession.objects.get(id=movie_session_id)
+
+            Ticket.objects.create(
+                order=order,
+                movie_session=movie_session,
+                row=row,
+                seat=seat
+            )
+
+        if date:
+            created_at = parse_datetime(date)
+            if created_at:
+                order.created_at = created_at
+                order.save()
+
+
+def get_orders(username: str = None) -> QuerySet:
+
+    if username:
+        user = User.objects.get(username=username)
+        return Order.objects.filter(user=user)
```

Comment:
Avoid making call to the user here, try to use ORM features to build the call from `Order` only